llama_stack/cli/stack/build.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, subparsers: argparse._SubParsersAction):
 
     def _add_arguments(self):
         self.parser.add_argument(
-            "config",
+            "--config",
```

Comment:
I think keeping this positional and having it be an optional `<name_or_yaml>` would be good. so the command line feels easier

llama stack build foo
llama stack build ~/foo/bar.yaml